### DISCUSSION TITLE
Reminders for content that's NOT updated

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1242] [Feature Requests]
> I'm looking for a solution that can send my authors a notification when a post hasn't been revised in say, 3 months. This to keep the content always up-to-date. Would this be possible with one of your plugins?

This shouldn't be too hard? Check for any post that doesn't have a new revision in the last 3 months?

**Comment by ghost** [comment]
We're interested in this feature as well. It could be similar to the "After the content is published" with the option to specify the Amount and Unit of time. Maybe something like "When content has not been updated". It would be helpful to have an additional option to repeat the notification into the future. For example, you might create a reminder that is triggered when pages haven't been revised in 52 weeks and the reminders are sent every 52 weeks into the future until the page is revised. That way, you can ensure each page on the site is reviewed at least once a year.

**Comment by stevejburge** [comment]
One more for this:

> We need to system to email a page author/department head when a page hasnt been updated for 180 days. We want this to apply to all existing pages, and if they are outside the 180 day period, the emails will be automatically sent for past items based on its published or updated dates.

**Comment by stevejburge** [comment]
> I'd like to schedule via the calendar a review cycle (defined within the task, could be any date/time). This way we're always reviewing each page in an appropriate timeframe to make sure content isn't stale, etc.
> 
> I tried the free version of publishpress, and there is a calendar, but the calendar event seems to only create content (a page, post, etc.). I want to create an event that acts as a review task, to keep us on a schedule.

**Comment by stevejburge** [comment]
> Just wondering if there is a way, through any of your Pro plugins (or some other) to identify current published pages that save haven't been updated in a 1 year from the publish date, and change their status so they need to be reviewed?

**Comment by stevejburge** [comment]
One more here:

> I want to set a date in the future for a specific page or post to trigger a notification that reminds you to update the content? I can see this is kind of possible within notifications itself by selecting the time in the future rather than a specific date. But is there a more simple way to do this when such as when viewing a post or page by just selecting a tick box to trigger a notification, select the date for it to trigger and who should receive it. Kind of like the publish date?

**Comment by stevejburge** [comment]
One more:

> I have a client who are a charity with a big team that changes from time to time. Because of this content can sometimes becomes stale because the "owner" has left or the content has been forgotten. I'd like to be able to set reminders on pages and posts which then email the owner or a team to review their content. This way they can keep it fresh or remove it if it losses relevance

**Comment by stevejburge** [comment]
> I wondered if I could use your plugin to plan for "When to update this content next."
> When we publish evergreen content, we want to check back in and update/revise it 6 or 12 month later.
> Besides being able to have admin comments in a post, I'd love to be able to set a future date for each post and then have an editorial calendar that reminds me when it's time to revisit a post.
